src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml

```diff
@@ -2641,4 +2641,10 @@
     <Left>net7.0/System.Runtime.Loader.dll</Left>
     <Right>net8.0/System.Runtime.Loader.dll</Right>
   </Suppression>
+  <Suppression>
```

Comment:
@ViktorHofer Even though I regenerated the suppression file, I'm still seeing related ApiCompat errors in CI. Do you know what might be at fault here?

Reply:
Responded offline in Teams but TL;DR:

> dotnet pack src\libraries\System.Text.Json\src /p:ApiCompatGenerateSuppressionFile=true

Reply:
Thanks!